sdk/batch/Microsoft.Azure.Batch/Tools/ObjectModelCodeGenerator/Spec/VMExtension.json

```diff
@@ -63,6 +63,17 @@
       },
       "Value": null
     },
+    {
+      "Key": {
+        "Type": "bool?",
+        "Name": "EnableAutomaticUpgrade",
+        "SummaryComment": "Indicates whether the extension should be automatically upgraded by the platform if there is a newer version of the extension available.",
+        "RemarksComment": null,
+        "BoundAccess": "read,write",
```

Comment:
Should this be read-only for BoundAccess? I didn't think this was updatable.

Reply:
Yes, this is not an updatable property, we should change it to read-only for BoundAccess. And I am also considering if we should change all properties in VMExtension model to read-only for BoundAccess...

Reply:
The UpdatePool API doesn't update VMExtension, set write for BoundAccess is useless...